test_server.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+import pytest
+import server as app
+import requests
+
+url = 'http://localhost:8080'
+
+def test_question():
+    r = requests.get(url+'/question/1')
+    assert r.status_code == 200
+
+def test_download():
+    assert requests.get(url+'/question/1/inputs.txt').status_code == 200
+
+def test_file_upload():
+    assert requests.post(url+'/check/1').status_code == 200   # givin a error
```

Comment:
you are not supplying a file for the API to check. You can do that with [requests](http://docs.python-requests.org/en/master/user/quickstart/#post-a-multipart-encoded-file)

Reply:
> you are not supplying a file for the API to check. You can do that with [requests](http://docs.python-requests.org/en/master/user/quickstart/#post-a-multipart-encoded-file)

can you guide more on it? i have checked the link but not getting any idea how i can post the file in that form

Reply:
ok. what have you tried? Is that in some PR?

Reply:
NO, 2 out of 3 test functions are working fine, but for `def test_file_upload()`, i am having no idea how I post a file to that URL, 
i have googled a lot already

Reply:
ok. what is it about the link I posted that is confusing you? I'm talking about [this link](http://docs.python-requests.org/en/master/user/quickstart/#post-a-multipart-encoded-file)